### PR TITLE
Replace references to cl-quil.clifford::extract-qubits-used

### PR DESCRIPTION
**Pull request by notmgsk**
For details: https://github.com/rigetti/quilc/pull/157

### CODE REVIEW

File: src/package.lisp [hunk 1]
```diff
@@ -381,6 +381,12 @@
    #:rewrite-arithmetic                 ; FUNCTION/TRANSFORMATION
    )
 
+  (:export
+   #:compress-qubits
+   #:relabel-rewiring
```
Comment (appleby) [inline src/package.lisp]: File name comment above `:export`? Type-of comments?

File: quilc-tests.asd [hunk 2]
```diff
@@ -9,6 +9,7 @@
   :depends-on (#:quilc
                #:fiasco
                #:uiop
+               #:alexandria
```
Comment (appleby) [inline quilc-tests.asd]: Good :eye:

File: app/tests/rpcq-tests.lisp [hunk 3]
```diff
@@ -46,3 +46,26 @@
...
+    (sleep 1)
+    (unwind-protect
+         (rpcq:with-rpc-client (client "tcp://127.0.0.1:5555")
+           (let* ((quil "PHASE(pi/2) 0
+H 0")
+                  (gateset (list "PHASE(pi/2) 0" "H 0"))
+                  (server-payload (make-instance 'rpcq::|RandomizedBenchmarkingRequest|
+                                                 :|quil| quil
+                                                 :|depth| 2
+                                                 :|seed| 52
```
Comment (appleby) [inline app/tests/rpcq-tests.lisp]: Nice seed.

File: app/tests/rpcq-tests.lisp [hunk 4]
```diff
@@ -46,3 +46,26 @@
                (setf mat1 (quil::scale-out-matrix-phases mat1 mat2))
                (is (quil::matrix-equality mat1 mat2)))))
       (bt:destroy-thread server-thread))))
+
+(deftest test-generate-rb-sequence ()
+  (declare (optimize (debug 3)))
```
[the remark this answers is not in the document]
Comment (notmgsk) [reply app/tests/rpcq-tests.lisp]: nope!

File: src/package.lisp (hunk 1 above)
Comment (notmgsk) [reply src/package.lisp]: >_> fine

Comment (notmgsk) [reply src/package.lisp]: I don't think the type-of comments are useful, so I'm ignoring those

File: app/tests/rpcq-tests.lisp (hunk 4 above)
[the remark this answers is not in the document]
Comment (appleby) [reply app/tests/rpcq-tests.lisp]: Lemme tell you about a little key-chord I like to call `C-u C-c C-c`.

File: app/tests/rpcq-tests.lisp (hunk 3 above)
Comment (notmgsk) [reply app/tests/rpcq-tests.lisp]: This is actually copied from a pyQuil test. I should make a note of that in the code.

File: app/tests/rpcq-tests.lisp (hunk 4 above)
[the remark this answers is not in the document]
Comment (notmgsk) [reply app/tests/rpcq-tests.lisp]: @ecpeterson !!!

File: src/package.lisp (hunk 1 above)
Comment (appleby) [reply src/package.lisp]: I kind of agree with you, but consistency is also nice for O.C.D. types such as myself. I was just pattern matching on the rest of the file.

File: app/tests/rpcq-tests.lisp (hunk 4 above)
[the remark this answers is not in the document]
Comment (ecpeterson) [reply app/tests/rpcq-tests.lisp]: haha, i've been trying to remember this chord for months. thank you!

[the remark this answers is not in the document]
Comment (notmgsk) [reply app/tests/rpcq-tests.lisp]: Context: @ecpeterson was telling me about this fabled command the other day, but neither of us could remember/learn what it was called or the key binding.

[the remark this answers is not in the document]
Comment (appleby) [reply app/tests/rpcq-tests.lisp]: Glad to be of service

File: src/package.lisp [hunk 5]
```diff
@@ -381,6 +381,7 @@
    #:rewrite-arithmetic                 ; FUNCTION/TRANSFORMATION
    )
 
+  ;; compressor.lisp
```
Comment (appleby) [inline src/package.lisp]: I think you misspelled `analysis/compress-qubits.lisp` ?

Comment (notmgsk) [reply src/package.lisp]: I guess this is the signal that I'm done for the day. Thanks for the eyes

Comment (appleby) [reply src/package.lisp]: Go home, put your feet up. You've earned it.

**Comment by notmgsk** [comment]
Hm. I thought I was being clever making a separate PR for complete testing of the endpoints. I made a separate branch, rebased it onto this, and made a PR. I thought it would just keep the git history of #287 but apparently I/git merged the changes without keeping the history. Nevermind. Closing this in favour of #287 since it includes everything from here.

**Pull request closed without merging**